keyboards/projectkb/signature87/signature87.h

```diff
@@ -0,0 +1,50 @@
+#pragma once
+
+#include "quantum.h"
+
+#define KNO KC_NO
+
+#define LAYOUT_all( \
```

Comment:
If this board supports standard ANSI and ISO TKL layouts, could you add these here as `LAYOUT_tkl_ansi`/`iso` and in rules.mk?

Reply:
This PCB does not support ISO

Reply:
How about just ANSI with no split right shift & backspace?

Reply:
Looks like:

```
Signature 87

    MCU: ARM
    Switch: MX (PCB mount)
    Compatibility:
    ANSI Only
    Tsangan/WKL (7u) & Standard (6.25u)
    Stepped Caps, split backspace, split right shift
    Screw-in or PCB mount stabilizers
    LEDs: Caps lock & num lock indicator, in-switch LED (support)
```

https://store.projectkeyboard.com/products/preorder-h87

Reply:
Also, I know there are some other TKL PCBs that support split Backspace and Right Shifts. I'm thinking I want to extend the number of community layouts for TKL, that layout among them.